### PR TITLE
[blog post] OSCON report

### PR DESCRIPTION
**Pull request by gr2m**
Fixes https://github.com/hoodiehq/camp/issues/18

Google Doc for simpler editing: 
https://docs.google.com/document/d/1kkw_0vcHOg4PruLdLJIaGzrM6OPqMYJGeipOVPqBJE4/edit?usp=sharing

### CODE REVIEW

File: _posts/2016-05-16-hoodie-at-oscon-austin-2016.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+## Talks
+
+Our favourite talk of the event was Saron’s [“What we take for granted: Examining the barriers to contributing to open source”](http://conferences.oreilly.com/oscon/open-source-us/public/schedule/detail/49367) about luck & privilege. We are huge fans of [her podcast](http://www.codenewbie.org/podcast) and amazing [Code Newbie community](http://www.codenewbie.org/), and it was great to finally meet her. We also met [Maria](https://twitter.com/LadyNaggaga) and [Stacy](https://twitter.com/bitchwhocodes), all three working at Microsoft, doing amazing work, what a dream team!
+
+![Microsoft’s Dream Team](/blog/images/201605/microsofts-the-best.jpg)
+> Gregor, Stacy, Maria and Saron at the CodeNewbie booth
+
+## The exhibition
+
+In the exhibition hall, fellow Offline Camp co-organiser [Teri from IBM](https://twitter.com/EventTeri) introduced us to Marvin the robot who really is full of himself thinking he could win schnick schnack schnuck against Gregor. Turns out he couldn’t :)
```
Comment (janl) [inline _posts/2016-05-16-hoodie-at-oscon-austin-2016.md]: Schnick schanck schnuck is "rock, paper, scissors" in English ;D

Comment (gr2m) [inline _posts/2016-05-16-hoodie-at-oscon-austin-2016.md]: ah yes, but the IBM team went with Marvin to Germany and people would teach him Schnick Schnack Schnuck and they all loved it :) But will fix it

**Comment by jennwrites** [comment]
I'm editing in the Google doc right now ☺️

**Comment by jennwrites** [comment]
The one thing I noticed throughout is that you should probably include last names for people identified, since some had them and others did not. Great job @gr2m!

**Comment by gr2m** [comment]
Updated the blog post and added last names

**Comment by jennwrites** [comment]
LGTM!

**Comment by gr2m** [comment]
LGTM ship it anytime you think is best :)

**Pull request merged**